### PR TITLE
Adding draft requirements page

### PR DESCRIPTION
**Pull request by Burhan-Q**
@capsulecorplab, please verify and let me know if there are any issues

### CODE REVIEW

File: content/en/Requirements/Hardware/Design_requirements_outline.adoc [hunk 1]
```diff
@@ -0,0 +1,254 @@
+
+---
+title: "Project Tetra Hardware Requirements Outline"
```
Comment (capsulecorplab) [inline content/en/Requirements/Hardware/Design_requirements_outline.adoc]: I would personally re-title this "Hardware Requirements Outline" as to remove the redundancy

File: content/en/Requirements/Hardware/Design_requirements_outline.adoc [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+ 
+	- specification or test
+  
+	- *leading asterisk denotes specification or test needed
+
+	- trailing question mark denotes that definitive requirement for specification or test is unknown?
+
+
+== Overall device requirements/criteria:
+....
```
Comment (capsulecorplab) [inline content/en/Requirements/Hardware/Design_requirements_outline.adoc]: Is there are reason for these empty block comments?

**Review by capsulecorplab — COMMENTED** [review]
`Requirements` shouldn't be its own top-level section; everything should be nested under Explanations, Tutorials, How-To Guides, or References. In this case, I would remove the `Requirements/` folder and place `Design_requirements_outline.adoc` under `References`, being that requirements are a type of [reference](https://documentation.divio.com/reference/).

**Comment by capsulecorplab** [comment]
I would also do a rebase and drop the merge commit, but that's just personal preference.

File: content/en/Requirements/Hardware/Design_requirements_outline.adoc (hunk 2 above)
Comment (Burhan-Q) [reply content/en/Requirements/Hardware/Design_requirements_outline.adoc]: Added for additional spacing in the final form, looked too crowded (in my eye) without.

**Pull request merged**